.circleci/config.yml

```diff
@@ -0,0 +1,23 @@
+version: 2
+
+jobs:
+  publish:
```

Comment:
We need to add jobs for running tests here, similar to https://github.com/w3f/polkadot-secure-validator/blob/master/.circleci/config.yml They should run always and publish should only run on semver tags when the tests have passed.